.gitmodules

```diff
@@ -32,3 +32,6 @@
 	url = https://github.com/alanz/ghc-mod.git
 
 
+[submodule "submodules/floskell"]
+	path = submodules/floskell
+	url = https://github.com/bubba/floskell
```

Comment:
What did you have to change in your fork?

Reply:
[Adjust version bounds and tweak it so it builds with earlier and later haskell-src-exts](https://github.com/ennocramer/floskell/pull/10). I've made a PR for the changes so that hopefully we can drop the reliance on the submodule, since it brings us one step further away from #626